config/config.js.template

```diff
@@ -37,8 +37,6 @@ config.img_upload = {
 config.services = {};
 /* Molecular database service, used only for internal purposes (peakChartData query) */
 config.services.moldb_service_host = "{{ sm_graphql_moldb_service_host }}";
-/* Public IP of the server that hosts molecule images */
-config.services.mol_image_server_host = "{{ sm_graphql_mol_image_host }}";
```

Comment:
@iprotsyuk why to move it?

Reply:
At some point we discussed getting rid of absolute URLs for molecular images. I implemented a fix below, and this config parameter became unused

Reply:
I already don't remember what was motivation for getting rid of absolute urls